go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/hashicorp/terraform-plugin-log v0.9.0
 	github.com/oapi-codegen/oapi-codegen/v2 v2.4.0
 	github.com/oapi-codegen/runtime v1.1.1
+	github.com/tmaxmax/go-sse v0.8.0
```

Comment:
Is there a reason for not using `github.com/r3labs/sse` which has more stars and contributors?

Reply:
Neither of these libraries allow a running SSE connection to be closed by the client.

I ended up using this one because it at least has the `ResponseValidator` callback that can be used to get access to the `io.ReadCloser` for the response data, which we can close when we want to shutdown the connection.